src/main/resources/application.yaml

```diff
@@ -12,3 +12,7 @@ management:
 spring:
   application:
     name: Blob Router Service
+
+service:
+  - name: bulkscan
+    sas-validity: ${SAS_TOKEN_VALIDITY:300} #In seconds
```

Comment:
In processor we actually never changed this value. Is there a point for us introducing extra flexibility? Let's keep it simple and less headache configuring charts, if we need in future - we can introduce customisations

Reply:
ok, I've set to 300 seconds for now (Same as  processor)